src/lib/utils.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
```

Comment:
suggestion(optional): maybe add some kind of notice here

 ```suggestion
// utilities from lodash, embedded to keep things simple
```

Reply:
Ah these aren't directly from lodash. I've done this bundling thing before and I copied these from another library of mine. 

You're right about comments though. I'll add some JSDocs so it's clear what these do